### PR TITLE
update to Healpix 3.70

### PR DESCRIPTION
**Pull request by ntessore**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ntessore** [comment]
I'm not sure why the Linux build is failing, could anyone test locally?

**Comment by ntessore** [comment]
I have tried to debug this to the best of my abilities. The segmentation fault seems to come from within `libm` of all places. The issue appears to be the same that is described in [this StackOverflow question](https://stackoverflow.com/questions/27322295/ld-preload-causing-segmentation-fault-in-dynamic-library-loader), even though the cause is different: The stack trace for this PR looks identical to that one, and I have also run with `LD_DEBUG` to verify that `fma` is the last symbol lookup before the crash.

```
2020-11-09T16:49:45.3024294Z      20479:	symbol=fma;  lookup in file=.libs/alm2map_cxx [0]
2020-11-09T16:49:45.3025055Z      20479:	symbol=fma;  lookup in file=/home/conda/feedstock_root/build_artifacts/healpix_cxx_1604940406704/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libhealpix_cxx.so.2 [0]
2020-11-09T16:49:45.3026077Z      20479:	symbol=fma;  lookup in file=/home/conda/feedstock_root/build_artifacts/healpix_cxx_1604940406704/_build_env/lib/libstdc++.so.6 [0]
2020-11-09T16:49:45.3026584Z      20479:	symbol=fma;  lookup in file=/lib64/libm.so.6 [0]
2020-11-09T16:49:45.3027724Z      20479:	binding file /home/conda/feedstock_root/build_artifacts/healpix_cxx_1604940406704/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libsharp.so.1 [0] to /lib64/libm.so.6 [0]: normal symbol `fma' [GLIBC_2.2.5]
2020-11-09T16:49:45.3029047Z /home/conda/feedstock_root/build_artifacts/healpix_cxx_1604940406704/work/conda_build.sh: line 23: 20479 Segmentation fault      (core dumped) LD_DEBUG=all .libs/alm2map_cxx nlmax=128 infile=test.alm 'outfile=!test.fits' nside=64 polarisation=true
```

In addition, valgrind also identifies `fma` as the last named symbol before the crash.

Not sure where to go from here. Feel free to close, I suppose.

**Comment by ntessore** [comment]
@conda-forge/healpix_cxx I have also tried to work around the issue by moving to CentOS 7 with glibc 2.17. I'm not sure how to finish the configuration though, i.e. adding a correct `conda_build_config.yaml` file. Help?

**Comment by ntessore** [comment]
Moving to CentOS 7 and glibc 2.17 seems to have resolved this bizarre issue at last.

**Comment by beckermr** [comment]
So I'm fine with requiring centos7. However, I'd be nice to understand why.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
 requirements:
   build:
-    - autoconf
-    - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - make  # [not win]
   host:
     - cfitsio
+    - openmp
```
Comment (beckermr) [inline recipe/meta.yaml:25]: This isn't right. See the conda-forge docs on openmp.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,37 +1,35 @@
...
   run:
     - cfitsio
+    - openmp
```
Comment (beckermr) [inline recipe/meta.yaml:28]: ditto here

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,37 +1,35 @@
...
 
 test:
   commands:
     - test -f ${PREFIX}/lib/libhealpix_cxx.a
-    - test -f ${PREFIX}/lib/libc_utils.a
-    - test -f ${PREFIX}/lib/libcxxsupport.a
-    - test -f ${PREFIX}/lib/libfftpack.a
```
Comment (beckermr) [inline recipe/meta.yaml:34]: Did upstream deprecate these static libs?

File: recipe/meta.yaml (hunk 1 above)
Comment (ntessore) [reply recipe/meta.yaml]: apologies, leftovers from debugging

File: recipe/meta.yaml (hunk 3 above)
Comment (ntessore) [reply recipe/meta.yaml:34]: libc_utils and libcxxsupport are no longer built, and fftpack has been replaced by the compiled-in pocketfft in [version 3.50](https://healpix.sourceforge.io/html/install.htm#install:libsharp:config)

**Pull request closed without merging**